extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -23,12 +23,22 @@
 ?>
 
 <h2 id="<?php echo esc_attr( $player_id ); ?>__title" class="jetpack-podcast-player__title">
-	<?php if ( ! empty( $track ) && isset( $track['title'] ) ) : ?>
+	<?php if ( ! empty( $track ) && isset( $track['title'] ) && isset( $track['link'] ) && isset( $track['src'] ) ) : ?>
```

Comment:
Optionally, this could be simplified into one `isset()` call:
```suggestion
	<?php if ( isset( $track['title'], $track['link'], $track['src'] ) ) : ?>
```

Reply:
Good point, addressed!